understanding/multiple-ways.html

```diff
@@ -44,7 +44,7 @@ <h2>メリット</h2>
                <li>複数の手段でサイトをナビゲートする機会を提供することによって、人々が情報をより早く見つけるのに役立つことができる。視覚障害のある利用者は、画面拡大ソフト又はスクリーンリーダーを用いて大きなナビゲーションバーを経由してスクロールするよりも、検索機能を使用してサイト内の適切な部分へナビゲートしていくほうが容易なことがある。認知障害のある人は、複数のウェブページを読んで横断するするよりも、サイト全体を見渡すことのできる目次又はサイトマップを好むことがある。中には、サイトのコンセプトやレイアウトを最もよく理解できるように、ウェブページからウェブページへと順番に移動しながらサイト内を探索するのを好む利用者もいるかもしれない。</li>
                
                
-               <li>認知に制約のある利用者は、階層構造を用いたナビゲーションスキームは分かりづらいことがあるため、検索機能を使うほうがより容易なことがある。</li>
+               <li>認知機能に制約のある人は、理解が難しいかもしれない階層的なナビゲーションスキームを使用するよりも、検索機能を使用する方が簡単だと見出すかもしれない。</li>
```

Comment:
```suggestion
               <li>認知機能に制約のある人は、理解が難しいかもしれない階層的なナビゲーションスキームを使用するよりも、検索機能を使用する方が簡単だと感じるかもしれない。</li>
```



> Individuals with cognitive limitations may find it easier to use search features than to use a hierarchical navigation scheme that may be difficult to understand.

https://eow.alc.co.jp/search?q=find+it+easier
find it easier to
～する方が楽だ［簡単だ］と思う［感じる］

Reply:
find の訳、「感じる」でよかったのですね。ありがとうございます。